shell/browser/browser_mac.mm

```diff
@@ -377,11 +377,14 @@ void RemoveFromLoginItems() {
   NSDictionary* options = DictionaryValueToNSDictionary(about_panel_options_);
 
   // Credits must be a NSAttributedString instead of NSString
-  id credits = options[@"Credits"];
+  NSString* credits = (NSString*)options[@"Credits"];
   if (credits != nil) {
+    NSAttributedString* creditString = [[NSAttributedString alloc]
+        initWithString:credits
+            attributes:@{NSForegroundColorAttributeName : [NSColor textColor]}];
+
     NSMutableDictionary* mutable_options = [options mutableCopy];
```

Comment:
Not a problem of this PR, but it should also have `autorelease` or be put in a `base::scoped_nsobject`.